aim/sdk/adapters/hugging_face.py

```diff
@@ -97,6 +96,13 @@ def on_log(self, args, state, control,
                 if log_name.startswith(prefix):
                     log_name = log_name[len(prefix):]
                     context = {'subset': prefix[:-1]}
+                    if '_' in log_name:
+                        sub_dataset = AimCallback.find_most_common_substring(
```

Comment:
lets restructure this code to make it more readable.

Reply:
Any recommendations on this?